task_test.go

```diff
@@ -64,4 +64,21 @@ from(bucket:"x")
 			t.Fail()
 		}
 	})
+	t.Run("test add new option", func(t *testing.T) {
```

Comment:
Is this test related to the new code?  Perhaps add a test using org name if it doesnt exist?

Reply:
Same question, this test doesn't seem related to this PR.

Reply:
Whoops, is related to another PR, the files got mixed.

Reply:
Its still a useful test, and it didn't get included in the merged pr it was supposed to, so I'll leave it in?